improve description in fastlane (#6)
CDrummond/squeezelite: fastlane/metadata/android/en-US/full_description.txt

> <p><i>Squeezelite</i> is a 'headless' audio player for Lyrion Music Server (LMS) - <a href='https://github.com/LMS-Community/slimserver'>Squeezelite</a>, ported to Android. You will need an LMS server in your local network to make use of this; <a href='https://lyrion.org/getting-started/'>see here</a> for getting started.</p>

Reviewer: ```suggestion
<p><i>Squeezelite</i> is a 'headless' audio player for Lyrion Music Server (LMS) - <a href='https://github.com/LMS-Community/slimserver'>Squeezelite</a>, ported to Android. You will need access to an LMS server from your network to make use of this; see <a href='https://lyrion.org/getting-started/'>Getting Started with LMS</a>.</p>
```
